Events/src/main/java/com/github/swcraftlyon/meetup/infra/dao/MangoEventDao.java

```diff
@@ -0,0 +1,41 @@
+package com.github.swcraftlyon.meetup.infra.dao;
+
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+
+
+public class MangoEventDao implements IEventDAO {
+
+    public static final String EVENT_DB = "EventDB";
+    public static final String MONGODB_LOCALHOST_27017 = "mongodb://localhost:27017";
```

Comment:
You got the idea I guess. If ApplicationConfig is too broad a dependency, you can have a MongoConfig.